pyt/cfg/stmt_visitor.py

```diff
@@ -810,7 +816,6 @@ def add_module(  # noqa: C901
         module_path = module[1]
 
         parent_definitions = self.module_definitions_stack[-1]
-        # The only place the import_alias_mapping is updated
```

Comment:
Nit: Because the `import_alias_mapping` code can be a little hard to reason about, maybe we can keep this comment by adjusting it to
```python
# Here and in `visit_Import` are the only places the `import_alias_mapping` is updated
```